doc/reference/configuration/configuration_reference.rst

```diff
@@ -2411,45 +2460,74 @@ The ``process`` defines configuration parameters of the Tarantool process in the
     | Type: boolean
     | Default: false
     | Environment variable: TT_PROCESS_BACKGROUND
-
+    | Dynamic: no
 
 .. _configuration_reference_process_coredump:
 
 .. confval:: process.coredump
 
+    **Since:** :doc:`3.0.0 </release/3.0.0>`.
+
+    Whether to create coredump files.
 
     |
     | Type: boolean
     | Default: false
     | Environment variable: TT_PROCESS_COREDUMP
+    | Dynamic: no
 
 .. _configuration_reference_process_title:
 
 .. confval:: process.title
 
-    Sets the name of the server's process title (what’s shown in the COMMAND column for
-    ``ps -ef`` and ``top -c`` commands).
+    **Since:** :doc:`3.0.0 </release/3.0.0>`.
+
+    Add the given string to the server's process title
+    (what’s shown in the COMMAND column for ``ps -ef`` and ``top -c`` commands).
+
+    For example, if you set ``title=''``, :samp:`ps -ef` shows
+    the Tarantool server process thus:
+
+    .. code-block:: console
+
+        $ ps -ef | grep tarantool
+        1000     14939 14188  1 10:53 pts/2    00:00:13 tarantool <running>
+
+    But if you set ``title='sessions'``, then the output looks like:
+
+    .. code-block:: console
+
+        $ ps -ef | grep tarantool
+        1000     14939 14188  1 10:53 pts/2    00:00:16 tarantool <running>: sessions
 
```

Comment:
`ps -ef` shows the following info on my machine. Not sure that we need such output in the config reference - looks like they might differ on different platforms. IMO, configuration examples can be more useful here.

```
% ps -ef | grep tarantool
  503  4330  4328   0 10:20AM ttys000    0:00.52 tarantool <running>: sessio -n instance001 -c /Users/john.doe/PycharmProjects/doc/doc/code_snippets/snippets/config/instances.enabled/application/config.yaml
```

If we still want to show the output, we can tell which platform is used for this example.

Reply:
Yes, ps and top have Mac-specific nuances. I guess saying "Linux commands" above would be enough to avoid confusion.